components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/BuildCmd.java

```diff
@@ -327,4 +334,22 @@ private void initTarget(String projectName) throws IOException {
         CmdUtils.createDirectory(projectModuleDir, true);
     }
 
+    /**
+     * Invoke call home.
+     *
+     */
+    private void invokeCallhome() {
+        String productHome = CmdUtils.getCLIHome();
+        String trustStoreLocation = productHome + File.separator + RESTServiceConstants.DEFAULT_TRUSTSTORE_PATH;
```

Comment:
Lets read the trust store from toolkit-config.toml

Reply:
fixed